processor/README.md

```diff
@@ -133,15 +133,29 @@ Use the token to authenticate requests protected by JWT: `Authorization: Bearer
 The processor exposes following endpoints to execute various operations with Adyen platform:
 
 ### Get payment methods
-TBC
+It obtains the available payment methods supported by the given Adyen account.
```

Comment:
to be honest, I have doubts about this endpoint as it is not used, it's mainly for the dropin and as the processor can't be used without the enabler, maybe it would be worthy to not expose it or at least, not document it.

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/104/commits/5299dd2354b51e7b3167a1d3b70e5670df77803a